examples/wasm/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Button example</title>
+    <style>
+      .clickable-area {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: 200px;
+        height: 100px;
+        background-color: lightgray;
+        user-select: none; /* Don't copy text */
+      }
+    </style>
+  </head>
+  <body>
+    <script type="module" src="index.js"></script>
+    <div class="clickable-area">
+      <span>Press Area</span>
+    </div>
```

Comment:
Let's change this to 
```suggestion
    <button class="clickable-area" type="button">Button</button>
```